glean-core/metrics.yaml

```diff
@@ -1100,3 +1100,21 @@ glean.health:
                   type: number
                 error_message:
                   type: string
+
+  init_count:
+    type: counter
```

Comment:
@chutten @travis79 The current approach relies on the storage (since we need to store the "init counter"). What if instead we use the "legacy telemetry" session id approach?

When Glean initializes, we generate a uuid and call that "session id". It's not persisted, but we'll slap it on the pings that need it when we assemble the ping, like we do for other glean metadata, with the exception that this doesn't have to be stored . And we can scope it to the health ping only for now, as you did.

This way we don't rely on previous runs for the proper count and have distinct ids after Glean starts, regardless of the potential bugs.

Reply:
I'm not sure what benefit your approach has over combining `client_id` and `glean.health.init_count` together. On the contrary, it prevents us from counting and ordering Glean sessions (since there's no relationship between your proposed pseudo-application-lifetime session ids). 

I suppose it would help if the db were found to be volatile _within_ a session, but so far we haven't seen evidence of that in the wild (unless data platform infra isn't telling us about pings that no longer satisfy the schema because all the db-resident client_info and ping_info couldn't be found during ping assembly).

Does that make sense, @Dexterp37 ?

Reply:
Ah-ha! You've fallen into my trap! Counting distinct session ids per client id requires a stable client_id, which presupposes stable storage, which would mean we'd not risk losing the count across inits : D

Reply:
Counting sessions with a counter _also_ implies stable storage, with the additional implications of using a user-lifetime metric.

With that said, if you and Travis agree this is the path, that's ok!